### PR TITLE
feat:Liked page and Watch later added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Please review my Liked page and Watch later functionality.
Thank you!

### CODE REVIEW

**Comment by ArpitPatni** [comment]
Preview link: https://patnishub.netlify.app/

File: src/Components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -35,4 +35,5 @@
 ul li:hover{
     color: black;
```
Comment (rahul31agrawal) [inline src/Components/Sidebar/Sidebar.css]: use color variable names

File: src/Pages/Homepage/Homepage.css [hunk 2]
```diff
@@ -41,8 +41,8 @@
   }
   
   .options{
-    height: 30px;
-    width: 30px;
+    height: 40px;
```
Comment (rahul31agrawal) [inline src/Pages/Homepage/Homepage.css:44]: use rems instead of px

**Review by rahul31agrawal — APPROVED** [review]
Hi Arpit,

1. The code looks clean & easily understandable.
2. the folder structure is good.
3. I liked the theme of your video library
4. optional suggestion : remove spaces at the bottom of the video card

Thanks

**Pull request merged**